### PR TITLE
Add MoleculeNet model evaluation script

### PR DESCRIPTION
**Pull request by mihaimorariu**
This PR addresses #211. Standard scaling is currently not implemented and will follow soon.

### CODE REVIEW

File: examples/molnet/train_molnet.py [hunk 1]
```diff
@@ -103,135 +237,109 @@ def main():
...
+    loss_fun = molnet_default_config[dataset_name]['loss']
+
     if task_type == 'regression':
-        loss_fun = regression_loss_fun
-        model = Regressor(predictor, lossfun=loss_fun, metrics_fun=metrics_fun,
-                          device=args.gpu)
-        # TODO(nakago): Use standard scaler for regression task
+        model = Regressor(predictor, lossfun=loss_fun,
+                          metrics_fun=metrics_fun, device=args.gpu)
+        # TODO: Use standard scaler for regression task
```
Comment (mottodora) [inline examples/molnet/train_molnet.py]: TODO(nakago): ?

**Comment by mottodora** [comment]
@corochann 
Can you add scaling function after this PR?

File: examples/molnet/train_molnet.py (hunk 1 above)
Comment (mihaimorariu) [reply examples/molnet/train_molnet.py]: Sorry, I used an older script (as a template) which contained that line. I will remove the comment.

**Comment by mihaimorariu** [comment]
@corochann I have almost finished the scaling function as well, so I will make a separate PR once it is completed.

**Comment by corochann** [comment]
Ok thanks!!

File: examples/molnet/train_molnet.py (hunk 1 above)
Comment (mihaimorariu) [reply examples/molnet/train_molnet.py]: It should be fixed now.

File: examples/molnet/train_molnet.py [hunk 2]
```diff
@@ -61,37 +49,183 @@ def __call__(self, atoms, adjs):
...
+        method: Method name. See `parse_arguments`.
+        n_unit: Number of hidden units.
+        conv_layers: Number of convolutional layers for the graph convolution
+                     network.
+        class_num: Number of output classes.
+    Returns:
+        An instance of the selected predictor.
+    """
+
+    predictor = None
```
[the remark this answers is not in the document]
Comment (mihaimorariu) [reply examples/molnet/train_molnet.py]: Thank you, I made the changes.

**Comment by corochann** [comment]
@mottodora Could you check?

**Pull request merged**